content/articles/statistical-hypothesis-testing-python-implementation/index.md

```diff
@@ -0,0 +1,488 @@
+---
+layout: engineering-education
+status: publish
+published: true 
+url: /statistical-hypothesis-testing-python-implementation/
+title: Statistical Hypothesis Testing With Python Implementation
+description: This tutorial is a comprehensive explanation and implementation of statistical hypothesis testing methods such as t-test, chi-squared test & ANOVA in python, It is well known that most of the implementation of these techniques are always in R, and most Data Scientists and Analysts who use python always shy away from it.
+author: qoyum-olatunde-yusuf
+date: 2021-06-16T00:00:00-11:00
+topics: [Statistical Analysis, Hypothesis Testing, Data Science, Python ]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/statistical-hypothesis-testing-python-implementation/hero.png
+ alt: statistic hypothesis testing with python cover image
+ 
+---
+Hypothesis is the act of making observations and generating probing questions from the observations made in an attempt to come to a conclusion. Hypothesis testing is the test of assumptions on a population sample. It involves checking out whether a hypothesis should be accepted or not. In the world of work, hypothesis testing has been of great importance in making worthy decisions off mere observations. Billions of data are being generated on a daily basis around the globe and this has made the relevance of hypothesis testing a stronghold in making business decisions.
+<!--more-->
+
+### Understanding Hypothesis Testing
+
+There are two types of hypothesis testing, namely:
+1. Null Hypothesis
+2. Alternate Hypothesis
+
+**Null hypothesis** is the initial assumption about an event (also referred to as ground truth), whereas alternate hypothesis is an assumption that counters the initial assumption"
+
+In a scenario whereby a septuagenarian woman has a swollen stomach or baby bump, our initial assumption is going to be that 
+**"She is pregnant"**.
+
+while our alternate hypothesis will be
+**"She is not pregnant"**.
+
+The null hypothesis is a statement believed to be true unless we can prove otherwise beyond a reasonable doubt.
+
+In trying to assess two [mutually exclusive](https://www.mathsisfun.com/data/probability-events-mutually-exclusive.html) facts on a certain sample of data. We will refer to H0 as our null hypothesis and H1 as our alternate hypothesis, there seem to be too many alternate names for H0(ground truth, initial assumption, prior assumption, e.t.c) whereas H1 is vaguely described.
+
+To carry out hypothesis testing, the first step is to form an initial assumption and label it as H0. The next step would be to collect all data samples available to support our hypothesis.
+
+Collect all the shreds of evidence and analyze the data, and make a decision whether to accept the H0 or reject it. 
```

Comment:
This can be merged with the previous sentence.

Reply:
done Sir